src/main/java/org/broadinstitute/hellbender/transformers/IUPACReadTransformer.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.hellbender.transformers;
+
+import org.broadinstitute.hellbender.utils.BaseUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+public class IUPACReadTransformer implements ReadTransformer {
```

Comment:
I would add a javadoc here that explains that this transformation is done in-place without copying.

Reply:
Furthermore I would probably also provide an explanation for why this is necessary with cram reads.

Reply:
Done.